lib/panel/dock.js

```diff
@@ -14,6 +10,11 @@ class PanelDock {
   constructor(delegate: Object) {
     this.element = document.createElement('div')
     this.subscriptions = new CompositeDisposable()
+
+    const React = require('react')
```

Comment:
Done in https://github.com/steelbrain/linter-ui-default/commit/74f9f139d2a85cef72e74408e3c26e6defdddee2